src/main/java/learn/field_agent/controllers/GlobalExceptionHandler.java

```diff
@@ -33,16 +35,12 @@
         return ErrorResponse.build(result);
     }
 
-    @ExceptionHandler(RuntimeException.class)
-    public ResponseEntity<ErrorResponse> handleException(RuntimeException ex) {
-        Result result = new Result();
-        result.addMessage("Sorry but your request returned no results.", ResultType.NOT_FOUND);
-        return ErrorResponse.build(result);
-    }
```

Comment:
This handler isn't needed as long as you handle `Exception` (as you're doing below).